### DISCUSSION TITLE
Should we release 0.72.3?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#82] [Patches]
## Released, please refer to #83 

Current Release: [0.72.2](https://github.com/facebook/react-native/releases/tag/v0.72.2)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of issues we're aware of

* https://github.com/facebook/react-native/issues/37871
* https://github.com/facebook/react-native/issues/38294

#### List of picks requested

1. https://github.com/facebook/react-native/pull/35803

#### Local commits to backport to main

1. https://github.com/facebook/react-native/commit/8ce471e2fa802cc50ff2d6ab346627cb5f6d79b4 (from 72.1)

**Comment by kelset** [comment]
from @oblador:
> I'm nominating this metro fix that solves a regression in type: module project:
> https://github.com/facebook/metro/commit/692293339e3b9f542f7b83c04bdccf46fef05766
> Not sure of how the release flow works for metro, but I'm guessing the fix needs to be back ported to the 0.76 branch and released as a patch. cc @huntie

**Comment by scrgh** [comment]
<img width="334" alt="Screen Shot 2023-07-11 at 11 24 18 PM" src="https://github.com/reactwg/react-native-releases/assets/6205717/50358329-6b61-4333-a82f-80c48e6ebc31">
I got this error after upgraded to 7.2.2

**Reply by kelset** [reply]
~~please open a dedicated issue~~
Dedicated issue -> https://github.com/facebook/react-native/issues/38294

**Comment by leymytel** [comment]
Hi @kelset !
Is this fix included in 0.72.2? https://github.com/facebook/react-native/commit/86b5b4cc34dfa8b8ec1852f2cd195361e1a9311d
I can't see it in the release notes.

**Reply by kelset** [reply]
yes: https://github.com/facebook/react-native/commit/ee8d5e0259ccdfcd5fb8b2d3f897bdcd9bfb6b4b

it's not in the release notes 'cause @cortinico marked it as internal

**Comment by billnbell** [comment]
Please https://github.com/facebook/react-native/pull/38247

This fixes React-ImageManager for react-native-svg etc.

**Reply by kelset** [reply]
it's not merged yet so we can't pick it.
